### ISSUE TITLE
allow to configure web port

### ISSUE THREAD

**Issue by hlas** [f18m/ha-addon-dnsmasq-dhcp-server#6]
On start, I get:

```
dnsmasq-dhcp[161]: DHCP, IP range 192.168.1.100 -- 192.168.1.199, lease time 12h
dnsmasq-dhcp[161]: DHCP, sockets bound exclusively to interface end0
2024/09/27 20:32:12 listen tcp :8080: bind: address already in use
s6-rc: info: service legacy-services successfully started
2024/09/27 20:32:12 [notice] 160#160: using the "epoll" event method
2024/09/27 20:32:12 [notice] 160#160: nginx/1.24.0
2024/09/27 20:32:12 [notice] 160#160: OS: Linux 6.6.31-haos-raspi
2024/09/27 20:32:12 [notice] 160#160: getrlimit(RLIMIT_NOFILE): 1073741816:1073741816
2024/09/27 20:32:12 [notice] 160#160: start worker processes
2024/09/27 20:32:12 [notice] 160#160: start worker process 199
[18:32:12] WARNING: Halt add-on
s6-rc: info: service legacy-services: stopping
2024/09/27 20:32:12 [notice] 160#160: signal 15 (SIGTERM) received from 156, exiting
2024/09/27 20:32:12 [notice] 199#199: exiting
2024/09/27 20:32:12 [notice] 199#199: exit
dnsmasq[161]: exiting on receipt of SIGTERM
2024/09/27 20:32:12 [notice] 160#160: signal 17 (SIGCHLD) received from 199
2024/09/27 20:32:12 [notice] 160#160: worker process 199 exited with code 0
2024/09/27 20:32:12 [notice] 160#160: exit
```

8080 port is used by another addon...

**Comment by f18m** [comment]
hi @hlas ,
this is a good point. I think I can implement this feature (port configurability) pretty quickly.
Moreover I'm going to change the default port to something less common

**Comment by f18m** [comment]
@hlas ,
can you try to:

1. go to your Addon dashboard in HA interface
2. click the top-right icon "Check for updates"
3. click the bottom-right  icon "addon library" 
4. check that under "Francesco Montorsi add-ons" you can see "Dnsmasq-DHCP BETA"
5. install that addon
6. verify your issue is fixed?

Thanks!

**Comment by hlas** [comment]
It works flawlessly!
Thanks

**Comment by f18m** [comment]
> It works flawlessly! Thanks

Ok thanks for the feedback. I've merged the changes and released version 1.2.0 so I suggest you to switch to the "stable channel" and uninstall the BETA version.
Closing this issue!
